spec/features/roles_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe 'assigning roles' do
+  context 'with admin privilege' do
+    it 'assigns one role to user' do
+      admin = create :user, admin: true
+      login_as admin
+      visit admin_users_url
+      # binding.pry
+      check 'user_master_of_ceremonies'
+      click_on 'Save'
+      expect(page).to have_text 'User has been updated.'
+      expect(admin.master_of_ceremonies).to eql true
```

Comment:
In a feature test, better to check on a user experience to see that this change has taken effect.